### ISSUE TITLE
Wachlist Sync Not Removing From Trakt

### ISSUE THREAD

**Issue by Migz93** [Taxel/PlexTraktSync#1026]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists

### The problem

When running a sync it appears that plextraktsync is not able to remove items from my Trakt watchlist. It logs that it is doing it but the item is never actually removed from my Trakt watchlist.
It is able to successfully add items, just not remove. I have tested this with other movies/shows and seeing the same thing.

"Samaritan" movie is the one I'm expecting to be removed.
Config section:
```
sync:
  plex_to_trakt:
    collection: true
    ratings: true
    watched_status: true
    watchlist: true
  trakt_to_plex:
    liked_lists: false
    ratings: true
    watched_status: true
    watchlist: false
```

### Error trace / logs

```text
https://pastebin.com/UxigKxca
(Had to pastebin due to length)
```

### Expected behavior

Expect that movie Samaritan which is no longer on my Plex watchlist is successfully removed from Trakt watchlist.

### Steps to reproduce the behavior

1. Remove Samaritan from my plex watchlist.
2. Run `plextraktsync sync --id 245517` (Which is the ratingKey specifically for Samaritan).
3. Review Trakt watchlist, confirm Samaritan movie is still there.
4. Also tried running `plextraktsync` to do full sync.
5. Review Trakt watchlist, confirm Samaritan movie is still there.

### Inspect of problematic items

```text
Inspecting 245517: <tmdb:629176:Movie:245517:Samaritan>
URL: https://app.plex.tv/desktop/#!/server/163cfe56f7b4320095ea66ff2e3e73c72b0814f4/details?key=/library/metadata/245517
Media.Type: 'movie'
Media.Guid: 'plex://movie/5e833bb566500c0041d29ae5'
Media.Guids: [<Guid:imdb://tt5500218>, <Guid:tmdb://629176>, <Guid:tvdb://133332>]
Audio: 'stereo', 'English (AAC Stereo)'
Video: 'h264'
Subtitles:
  Subtitle 1: (English) None (codec: srt, selected: True, transient: None)
  Subtitle 2: (Português) None (codec: srt, selected: False, transient: None)
Parts:
  Part 1: /mnt/user/media/Movies/Movies/Samaritan (2022) /Samaritan (2022) [WEBDL-1080p][EAC3 5.1]-ADWeb.mkv
Guids:
  Guid: <PlexGuid:tmdb://629176>, Id: 629176, Provider: 'tmdb'
  Guid: <PlexGuid:tvdb://133332>, Id: 133332, Provider: 'tvdb'
  Guid: <PlexGuid:imdb://tt5500218>, Id: tt5500218, Provider: 'imdb'
Metadata: {'collected_at': '2022-08-26:T18:04:24.000Z', 'media_type': 'digital', 'resolution': 'hd_1080p', 'audio': 'dolby_digital_plus', 'audio_channels': '5.1'}
Trakt: https://trakt.tv/movies/473232
Plex Rating: None
Trakt Rating: None
Watched on Plex: False
Watched on Trakt: False
Plex play history:
```

### Install method

docker-compose

### Version

0.21.1

### Python Version

3.10.6

### Operating System and Version

Unraid 6.10.3 - ghcr.io/taxel/plextraktsync:latest

**Comment by twolaw** [comment]
Fix:
- https://github.com/Taxel/PlexTraktSync/pull/1027

**Comment by glensc** [comment]
> https://pastebin.com/UxigKxca
> (Had to pastebin due to length)

what length? I think you can upload .txt files to github issues

**Comment by Migz93** [comment]
> Fix:
> 
> * [batch=True for remove_from_trakt_watchlist #1027](https://github.com/Taxel/PlexTraktSync/pull/1027)

Thank you for the really quick fix.




> > https://pastebin.com/UxigKxca
> > (Had to pastebin due to length)
> 
> what length? I think you can upload .txt files to github issues

I first tried with just pasting that entire log from the pastebin into the "Error trace / logs" section but when i tried to submit the issue I got some error about body length being too long so I went back and changed that to pastebin.
Just tested and although I can't upload the log file into the "Error trace / logs" I can just upload it straight into "The Problem" section which is handy to know and what I'll use in future if needed, thank you.
